examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
`test/images/network-tester/rc.json` and `test/images/network-tester/service.json` are [explicitly included in the test tarball](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh#L90-L91). We should probably do something similar for `test/e2e/testing-manifests`.